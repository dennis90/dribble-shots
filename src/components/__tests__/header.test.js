import React from 'react'
import {shallow} from 'enzyme'

import Header from '../header'

describe('Document Header Component', () => {
  it('should render document header component', () => {
    const wrapper = shallow(<Header />)
    expect(wrapper).toMatchSnapshot()
  })
})
