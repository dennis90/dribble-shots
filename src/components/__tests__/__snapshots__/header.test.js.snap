// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Document Header Component should render document header component 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Header />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <span
          className="app-title"
>
          Dribbble
</span>,
        <span
          className="app-subtitle"
>
          SHOTS
</span>,
      ],
      "className": "header",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "Dribbble",
          "className": "app-title",
        },
        "ref": null,
        "rendered": "Dribbble",
        "type": "span",
      },
      Object {
        "instance": null,
        "key": null,
        "nodeType": "host",
        "props": Object {
          "children": "SHOTS",
          "className": "app-subtitle",
        },
        "ref": null,
        "rendered": "SHOTS",
        "type": "span",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <span
            className="app-title"
>
            Dribbble
</span>,
          <span
            className="app-subtitle"
>
            SHOTS
</span>,
        ],
        "className": "header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "Dribbble",
            "className": "app-title",
          },
          "ref": null,
          "rendered": "Dribbble",
          "type": "span",
        },
        Object {
          "instance": null,
          "key": null,
          "nodeType": "host",
          "props": Object {
            "children": "SHOTS",
            "className": "app-subtitle",
          },
          "ref": null,
          "rendered": "SHOTS",
          "type": "span",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
